spec/_widget/components/search.spec.js

```diff
@@ -1,4 +1,5 @@
-import { search, prepareArticles, dictionaryTermMatches, articleScore, fixRelativeImgSrcs } from '../../../_widget/src/components/app/search.js';
+import Search from '../../../_widget/src/components/app/search.js';
+import ARTICLES from '../../../output/search.json';
```

Comment:
There's a trade-off here in that we need to compile the static site before running the specs... but we get great data to work with.